library/include/modules/Units.h

```diff
@@ -180,6 +181,10 @@ DFHACK_EXPORT std::string getCasteProfessionName(int race, int caste, df::profes
 DFHACK_EXPORT int8_t getProfessionColor(df::unit *unit, bool ignore_noble = false);
 DFHACK_EXPORT int8_t getCasteProfessionColor(int race, int caste, df::profession pid);
 
+DFHACK_EXPORT df::goal_type getGoalType(df::unit *unit, int goalIndex = 0);
+DFHACK_EXPORT std::string getGoalName(df::unit *unit, int goalIndex = 0);
+DFHACK_EXPORT bool isGoalAchieved(df::unit *unit, int goalIndex = 0);
```

Comment:
If you could also add these to the units module in `LuaApi.cpp` (and maybe even document them in `docs/Lua API.rst`), that would be great.

Reply:
Thanks, I'll look to add these tomorrow.

Reply:
There's a hopefully-helpful page about building the docs here, if you want to see how they look when built: https://docs.dfhack.org/en/latest/docs/Documentation.html
It could be a lot of overhead relative to the size of your changes, so I'd also be willing to build them and make sure the formatting looks ok.